packages/devtools_app/lib/src/screens/app_size/app_size_table.dart

```diff
@@ -13,19 +13,27 @@ import '../../ui/colors.dart';
 
 import 'app_size_controller.dart';
 
-const deferredLoadingSupport = false;
+//temporary feature flag for deferred loading
+const deferredLoadingSupport = true;
 
 class AppSizeAnalysisTable extends StatelessWidget {
   factory AppSizeAnalysisTable({
     required TreemapNode rootNode,
     required AppSizeController controller,
   }) {
-    final treeColumn = _NameColumn(currentRootLevel: rootNode.level);
+    final treeColumn = _NameColumn(
+      currentRootLevel:
+          deferredLoadingSupport ? rootNode.children[0].level : rootNode.level,
```

Comment:
For this (and below) we should be checking if the data we have is for a deferred app, not whether the feature is enabled (so `controller.isDeferredApp`);